app/api/v2/models/orders_model.py

```diff
@@ -0,0 +1,151 @@
+                     "current_location": parcels[6],
+                     "destination": parcels[7],
+                     "price": parcels[8],
+                     "status": parcels[9],
+                     }
+        return my_parcel
+
+    @staticmethod
+    def delete_order(parcel_id):
+        """delete an order"""
+        db.cursor.execute("DELETE from parcels WHERE order_id = %s ", (parcel_id,))
+
+    @classmethod
+    def cancel_order(cls, parcel_id):
+        """cancel an order only if its not delivered"""
+        try:
+            order = cls.cancelled_or_delivered(parcel_id)
+            if order:
+                db.cursor.execute("UPDATE parcels SET status=%s WHERE parcel_id = %s",
+                                  ('cancelled',
+                                   parcel_id))
+                db.commit()
+                return {"message": "order cancelled successfully"}
+            return {"message": "order is either cancelled or already delivered"}
+        except Exception as e:
+            return {"Message": e}
+
+    @classmethod
+    def update_destination(cls, parcel_id, destination):
+        """change order destination"""
+        order = cls.cancelled_or_delivered(parcel_id)
+        if order:
+            db.cursor.execute("""UPDATE parcels SET destination =%s WHERE parcel_id = %s""", (destination,
+                                                                                              parcel_id))
+            db.commit()
+            return {"message": "order updated successfully"}
+        return {"order is either cancelled or already delivered"}
+
+    @classmethod
+    def change_location(cls,parcel_id,current_location):
+        order = cls.cancelled_or_delivered(parcel_id)
+        if order:
+            db.cursor.execute("""UPDATE parcels SET current_location =%s WHERE parcel_id = %s""", (current_location,
+                                                                                              parcel_id))
+            db.commit()
+            return {"message":cls.get_single_order(parcel_id)}
+
+    @staticmethod
+    def check_exists(parcel_id):
+        try:
+            db.cursor.execute("SELECT * FROM parcels WHERE parcel_id = %s ", (parcel_id,))
+            if db.cursor.fetchone() is not None:
+                return True
+        except Exception as e:
+            return {"Message": e}
+
+    @staticmethod
+    def cancelled_or_delivered(parcel_id):
+        db.cursor.execute("SELECT * FROM parcels WHERE parcel_id = %s AND status = 'on-transit'",
+                          (parcel_id,))
+        order = db.cursor.fetchone()
+        if order:
+            return True
+
+    @classmethod
+    def get_by_specific_user(cls, user_id):
+        """"""
+        try:
+            db.cursor.execute("SELECT * FROM parcels WHERE user_id = %s ",(user_id))
+            db.cursor.dictfetchall()
+        except Exception as e:
+            return {"Message": e}
+
```

Comment:
You should leave only one line space at the end of each file, not more than one.